services/blockchain-app-registry/shared/utils/fsUtils.js

```diff
@@ -51,17 +51,25 @@ const mkdir = async (directoryPath, options = { recursive: true }) => {
 	);
 };
 
-const rmdir = async (directoryPath, options = { recursive: true }) => {
+const rmdir = async (directoryPath, options = { recursive: true }) => new Promise((resolve) => {
 	logger.debug(`Removing directory: ${directoryPath}`);
-	await fs.rmdir(
+	fs.rm(
 		directoryPath,
-		options,
+		{
+			...options,
+			recursive: true,
+		},
```

Comment:
This is incorrect. Options must be passed as is, like it was before. This will override user's instruction for `recursive: false` as well.

Reply:
I think this is fine. `fs.rm` only deletes files by default. Since the function is for deleting directory, we must pass `recursive: true` to actually delete it regardless of what parameters have been passed.